.circleci/config.yml

```diff
@@ -22,9 +22,9 @@ references:
       run:
         name: Create requirement files
         command: |
-          pip install pipfile-requirements
-          pipfile2req > requirements.txt
-          pipfile2req -d > requirements-dev.txt
+          pip install pipenv
```

Comment:
let's wait for https://github.com/demisto/demisto-sdk/pull/1963/ to be approved, then merge it into this one and proceed